runtime/libpgmath/lib/common/kqnint.c

```diff
@@ -0,0 +1,22 @@
+/*
+ * Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+ * See https://llvm.org/LICENSE.txt for license information.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ */
+
+/* Intrinsic function which take quad precision arguments. */
+
+#include "mthdecls.h"
+/* INT */
+#define LONG_LONG_MAX 9223372036854775807
+long long __mth_i_kiqnint(long double d)
+{
+  if (d > 0 && (d > LONG_LONG_MAX))
+    return LONG_LONG_MAX;
+
+  if (d < 0 && (d < -LONG_LONG_MAX - 1))
```

Comment:
Thanks, because the code already commited to master branch, I filed a new PR https://github.com/flang-compiler/flang/pull/1132 to fix the format issue.